book/routing.rst

```diff
@@ -1162,11 +1162,11 @@ each separated by a colon:
 
 For example, a ``_controller`` value of ``AppBundle:Blog:show`` means:
 
-=========  ==================  ==============
-Bundle     Controller Class    Method Name
-=========  ==================  ==============
-AppBundle  ``BlogController``  ``showAction``
-=========  ==================  ==============
+=============  ==================  ==============
+Bundle         Controller Class    Method Name
+=============  ==================  ==============
+``AppBundle``  ``BlogController``  ``showAction``
```

Comment:
I agree to make this change for the class name and method name. However, do we do the same with bundle names somewhere else?